source/extensions/tracers/datadog/agent_http_client.h

```diff
@@ -67,12 +67,13 @@ class AgentHTTPClient : public datadog::tracing::HTTPClient,
    * @param on_response callback to invoke when a complete response is received
    * @param on_error callback to invoke when an error occurs before a complete
    * response is received.
+   * @param deadline
```

Comment:
Once we figure out how/if the timeout is used by the underlying HTTP implementation, we can accurately describe in this contract what `deadline` means.